testsuite/systemtest_long/procgen/ProGe/tcetest_generate_debugger.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+### TCE TESTCASE
+### title: Generate a processor with debugger interface and simulate
+
+TCECC=../../../../tce/src/bintools/Compiler/tcecc
+PROGE=../../../../tce/src/procgen/ProGe/generateprocessor
+PIG=../../../../tce/src/bintools/PIG/generatebits
+GENBUSTRACE=../../../../tce/tools/scripts/generatebustrace.sh
+
+ADF=data/debugger/debugger.adf
+IDF=data/debugger/debugger.idf
+TPEF=debugger.tpef
+DBIF=data/debugger/debugger_if-pkg.vhdl
+SRC=data/debugger/fib.c
+PROC=tb/proc_arch.vhdl
+PROGE_OUT=proge-output
+TBBUSTRACE=$PROGE_OUT/execbus.dump
+TTABUSTRACE=ttabustrace
+
+function eexit {
+    echo $1
+    exit 1
+}
+
+function clear_test_data {
+  rm -f *.img
+  rm -rf $PROGE_OUT
+  rm -f *.tpef
+  rm -f $TTABUSTRACE
+}
+
+clear_test_data
+$TCECC -a $ADF -o $TPEF $SRC || eexit "Compilation failed with $SRC"
```

Comment:
Hint: you can use TCE binaries without relative path to them in the test scripts (e.g. just "tcecc -a $ADF ...").